proxy/envoy/watcher.go

```diff
@@ -232,10 +233,22 @@ func runEnvoy(mesh *proxyconfig.ProxyMeshConfig, node, configpath string) proxy.
 
 			glog.V(2).Infof("Envoy command: %v", args)
 
+			logFile, errLogFile := os.OpenFile(mesh.ErrorLogFile, os.O_RDWR|os.O_APPEND, 0666)
```

Comment:
Well, that makes things worse - systemd will pass a pipe, so "/dev/stderr" can't be opened. Let's not mess stderr.